pkgs/development/interpreters/wasmer/default.nix

```diff
@@ -8,25 +8,25 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "wasmer";
-  version = "0.17.0";
+  version = "1.0.2";
 
   src = fetchFromGitHub {
     owner = "wasmerio";
     repo = pname;
     rev = version;
-    sha256 = "05g4h0xkqd14wnmijiiwmhk6l909fjxr6a2zplrjfxk5bypdalpm";
+    sha256 = "0ciia8hhkkyh6rmrxgbk3bgwjwzkcba6645wlcm0vlgk2w4i5m3z";
     fetchSubmodules = true;
   };
 
-  cargoSha256 = "1ssmgx9fjvkq7ycyzjanqmlm5b80akllq6qyv3mj0k5fvs659wcq";
+  cargoSha256 = "08r2b4s005w8r207jwq2fd43y3prgd8pg1m72aww1r7yrbxdr0v2";
 
   nativeBuildInputs = [ cmake pkg-config ];
 
   # Since wasmer 0.17 no backends are enabled by default. Backends are now detected
   # using the [makefile](https://github.com/wasmerio/wasmer/blob/master/Makefile).
   # Enabling cranelift as this used to be the old default. At least one backend is
   # needed for the run subcommand to work.
-  cargoBuildFlags = [ "--features" "backend-cranelift" ];
+  # cargoBuildFlags = [ "--features" "backend-cranelift" ];
```

Comment:
Does this not mean that no backends are enabled anymore?

Reply:
`cargoBuildFlags = [ "--features" "backend-cranelift" ];` unfortunately fails with:

```
error: Package `wasmer-workspace v1.0.2 (/private/tmp/nix-build-wasmer-1.0.2.drv-0/source)` does not have the feature `backend-cranelift`
```

It appears back-ends are enabled by default; the documentation says to simply run `make build-wasmer`: https://docs.wasmer.io/ecosystem/wasmer/building-from-source.

The Makefile enables cranelift by default [here](https://github.com/wasmerio/wasmer/blob/9e0122e1f7bdb6fedf5fc17cc9a3a0fda30031a0/Makefile#L115).

This build can also be successfully run with `cargoBuildFlags = [ "--all-features" ];`, which may be the better approach?

Reply:
How big the overhead of just enabling all features?

Reply:
Good question - see comment below